### DISCUSSION TITLE
How do I make Tailwind read the classes of an object that comes from a database?

### DISCUSSION THREAD

**Discussion by vizardkill** [tailwindlabs/tailwindcss#8315] [Help]
Let me explain, I have an application where I store html templates, and each template can contain N tailwind classes, the question is how do I make tailwind read the template once obtained from the database to load the classes?

Right now I use next js and load the data from the SSR

**Comment by vizardkill** [comment]
up

**Comment by carlos-dubon** [comment]
This wont work:
```jsx
const [databaseColor, setDatabaseColor] = useState("red")

class={`bg-${databaseColor}-500`}
```

This will work:
```jsx
const [databaseColor, setDatabaseColor] = useState(null)

// wrap inside useEffect to call only once 
setDatabaseColor("bg-red-500")

class={`${databaseColor ? databaseColor : "bg-blue-500" }`}
```


It won't work if you don't pass the entire class name!
